board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
That way there is some disagreement with model and default_config:
MODEL=          orangepi-plus-2e
BOARD_CONFIG=   orangepi_plus2e_defconfig